OPHD/Constants/Numbers.h

```diff
@@ -8,18 +8,18 @@
  */
 namespace constants
 {
-	const int FADE_SPEED = 300;
+	inline constexpr int FADE_SPEED{ 300 };
```

Comment:
Btw, `inline` is implied by `constexpr`. You only need to specify `constexpr`.

Reply:
That's not what the article I was reading said, but reading the actual standard does confirm this. Wherps. Oh well, I'll worry about it another time. :D